shared/javascripts/glyph.js

```diff
@@ -49,10 +49,11 @@ if (Privly === undefined) {
 
   // CommonJS Module
   if (typeof module !== "undefined" && module.exports) {
-    module.exports.glyph = Privly.glyph;
```

Comment:
I've also changed things so that it's now like this --
```
// load dependencies
.. require statements ..
// export interfaces
.. export statements ..
```
or else mixing up the statements can be confusing to look at.